examples/instances.rs

```diff
@@ -84,15 +91,47 @@ impl<const N: usize, const A: usize, F: PrimeField> StepCircuit<A, F> for Instan
     }
 }
 
+fn get_or_create_commitment_key<C: CurveAffine>(
```

Comment:
I just don't want put `FOLDER` constant inside main codebase, but 'examples' haven't got any common modules. 

Because of that a lot of repeating of this fn